### DISCUSSION TITLE
3Q0907530Q dataset help

### DISCUSSION THREAD

**Discussion by Mex77** [jilleb/MQB-FPA#86] [Q&A]
Flashed gateway to 3Q0907530Q and there are no dataset around for Passat Alltrack 2017 2.0tsi with DCC..
Found this one (see .zip attached) for Golf and flashed it with odis.

Does anybody have a dataset that fits my car?

Could somebody help me?

[19_3Q0907530Q_OE_Golf_2019.zip](https://github.com/jilleb/MQB-FPA/files/10899500/19_3Q0907530Q_OE_Golf_2019.zip)

**Comment by Mex77** [comment]
Slider (yes) with Race?
But am I missing something?
Would like to have belt-pre and rear diff in Individual and Race..

[19_3Q0907530Q_RACE_20230307_140546_ODIS.zip](https://github.com/jilleb/MQB-FPA/files/10909659/19_3Q0907530Q_RACE_20230307_140546_ODIS.zip)

**Reply by jilleb** [reply]
Looks like there a few bytes too much in that...
I took the relevant bytes from the file, and parsed it. If you want belt-pre and rear diff to show up make sure you add them here:
![image](https://user-images.githubusercontent.com/8352494/223471838-d346e801-3dd4-41b0-a5e9-84a470f28947.png)

Rear diff already seems to be allright, but Automatic Belt Pre-Tensioning doesn't have any profiles yet. And did you do the long coding of the Gateway to enable these controls? Otherwise they will stay hidden.


Looking some more into your file... request values for DCC Slider are missing. Make sure it has all the values like you currently have at Steering.

Edit: It seems you inserted Rear diff into the list at the top, shifting all the controls down... it's better to add a new control to the end of the list.

**Comment by Mex77** [comment]
I cannot open 19_3Q0907530Q_RACE_20230307_140546_ODIS.xml on http://www.mqbtools.nl/fpa/..
Getting "Incorrect file size"..
If I could, it would be much easier 👍

**Comment by Mex77** [comment]
Thank you for helping me understand :)
The dataset is from a Golf, so I guess that alot is missing..
The FPA file is from 3Q0907530A* Passat, but don`t know what it is missing..

Many missing links here 🐙

**Reply by jilleb** [reply]
I can make a corrected one for you. The Odis file you sent has indeed the wrong size. I guess something went wrong on the site when you saved it because it has some additional data added to it.

**Comment by Mex77** [comment]
It was B80 file converted to 2388..
Used the latest one, with all the choices..

Thank you..

**Comment by jilleb** [comment]
[FPA_DATASET_ODIS_PC00_2303071922.zip](https://github.com/jilleb/MQB-FPA/files/10913226/FPA_DATASET_ODIS_PC00_2303071922.zip)

Try this ;-)

**Comment by Mex77** [comment]
Thank you :)
A little easier now..
But how do I get the DCC on the hardest in Sport/Race mode?
Tried sand, sport, race and unknown 5, but nothing gets close to "15" in Individual..
[19_3Q0907530Q_RACE3_ODIS_PC00.zip](https://github.com/jilleb/MQB-FPA/files/10919681/19_3Q0907530Q_RACE3_ODIS_PC00.zip)

**Reply by dvw-r** [reply]
I deleted my reply, I misunderstood your question.
My bad :)

EV Off = 15

**Comment by Mex77** [comment]
Ok, got it.. Will try it.. 

Thank you, again.. :)

**Comment by Mex77** [comment]
The alternator doesn't charge when driving lights are off.. So when it's "bright" I have to turn the switch (turn on driving lights) to get the alternator to charge again.. Is this a dataset, fpa-part-dataset or softcoding problem?

**Reply by Mex77** [reply]
Bright = daylight
